app/routes/app/$mType/m/$laborMarketAddress.sr/$serviceRequestId.review.tsx

```diff
@@ -29,27 +30,55 @@ export const loader = async ({ params }: DataFunctionArgs) => {
   return typedjson({ serviceRequest }, { status: 200 });
 };
 
-const validator = withZod(ClaimToReviewContractSchema);
+const claimToSubmitMachine = createBlockchainTransactionStateMachine<ClaimToReviewContract>();
+
+const validator = withZod(ClaimToReviewFormSchema);
 
 export default function ClaimToReview() {
   const { serviceRequest } = useTypedLoaderData<typeof loader>();
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
 
-  const [modalData, setModalData] = useState<{ data?: ClaimToReviewContract; isOpen: boolean }>({ isOpen: false });
+  const [state, send] = useMachine(claimToSubmitMachine, {
+    actions: {
+      notifyTransactionWait: (context) => {
+        // Link to transaction? https://goerli.etherscan.io/address/${context.transactionHash}
```

Comment:
This comment keeps following us around lol. Can you remove it from here and everywhere else?